cron/daily/charge_subscriptions.js

```diff
@@ -61,26 +61,26 @@ async function run(options) {
   );
 
   if (data.length > 0) {
-    await json2csv({ data, fields: csvFields }, async (err, csv) => {
-      vprint(options, 'Writing the output to a CSV file');
-      if (err) console.log(err);
-      else {
-        if (options.dryRun) {
-          fs.writeFileSync('charge_subscriptions.output.csv', csv);
-        } else {
-          if (!options.dryRun) {
-            vprint(options, 'Sending email report');
-            const attachments = [
-              {
-                filename: `${new Date().toLocaleDateString()}.csv`,
-                content: csv,
-              },
-            ];
-            await emailReport(start, orders, groupProcessedOrders(data), attachments);
-          }
+    vprint(options, 'Writing the output to a CSV file');
+    try {
+      const csv = json2csv({ data, fields: csvFields });
```

Comment:
Looks like the arguments are not matching the new API.

Reply:
Okay, fixed.